client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -189,5 +230,29 @@ export default class AccountOrders extends Vue {
     this.endDate = date;
     this.dateChanged();
   }
+
+  changeKeyword(value: string) {
+    const listConfig = { ...this.ordersList.listConfig };
+    listConfig.filters = { ...this.ordersList.listConfig.filters, keyword: value };
+    this.setListConfig(listConfig);
+  }
+
+  filtersChanged(activeFilters: string[]) {
+    activeFilters.length != this.filters.length ? this.allFiltersSelected = false : this.allFiltersSelected = true;
+    const listConfig = { ...this.ordersList.listConfig };
+    listConfig.filters = { ...this.ordersList.listConfig.filters, statuses: activeFilters };
+    this.setListConfig(listConfig);
+  }
+
+  toggleAllFilters(checked: string[]) {
```

Comment:
Rename to something like "toggleAllStatuses". "checked" is not a better name for param also IMHO.